draft-ietf-webtrans-http2.md

```diff
@@ -328,7 +328,7 @@ WT_RESET_STREAM can discard any data that it already received on that stream.
 ~~~
 WT_RESET_STREAM Frame {
   Type (i) = 0x04,
-  Length (i) = 0x00,
+  Length (i),
```

Comment:
So why swap padding vs. the rest of these? (And/or what is this intended to convey, I could just have it backwards)

Reply:
I should have removed all of them.  Length isn't zero for any frame now. (I added it to WT_PADDING because PADDING in QUIC doesn't have any body, but this one is different.)

Reply:
Gotcha, makes sense, thanks